components/inquiry_questionnaire/index.coffee

```diff
@@ -8,17 +9,21 @@ analytics = require './analytics.coffee'
 openErrorFlash = require './error.coffee'
 { steps, decisions, views } = require './map.coffee'
 
-module.exports = ({ user, artwork, inquiry, bypass }) ->
+module.exports = ({ user, artwork, inquiry, bypass, state_attrs }) ->
   { collectorProfile } = user.related()
   { userInterests } = collectorProfile.related()
 
   user.approximateLocation()
 
   # Allow us to trigger individual steps for debugging
   # by passing the named step as a `bypass` option
-  steps = [bypass] if bypass
+  steps = bypass if bypass
+
+  state = new State _.extend {
+    steps,
+    decisions
+  }, state_attrs
```

Comment:
We want to be able to optionally pass in additional state attributes for bypassing initial state purposes.